airflow/serialization/serializers/timezone.py

```diff
@@ -69,4 +82,28 @@ def deserialize(classname: str, version: int, data: object) -> Timezone:
     if isinstance(data, int):
         return fixed_timezone(data)
 
+    if "zoneinfo.ZoneInfo" in classname:  # capturing backports and stdlib
+        if PY39:
+            from zoneinfo import ZoneInfo
+        else:
+            from backports.zoneinfo import ZoneInfo
```

Comment:
Maybe we could use "old fashioned" try: except ? I'm biased towards to imports from `airflow` package because time to time it cause Circular Import Error

```suggestion
        try:
            from zoneinfo import ZoneInfo
        except ImportError:
            from backports.zoneinfo import ZoneInfo
```

And could we move it in to the root of the module? Or is it quite expensive import?

Reply:
In serialization we want to keep imports to a minimum, so not in the root indeed. I could check for the version explicitly like `sys.version_info >= (3, 9)`. The old fashioned stuff is slower (https://stackoverflow.com/questions/2522005/cost-of-exception-handlers-in-python) and in serialization the focus is on speed, particularly with datetimes.

Reply:
My favorite subject 🤣 The difference less than 1 microsecond for try/except and in the other case we have an application which communicate with DB and time to time this communications on suboptimal level (a lot of idle in transactions). But to be honest examples in this SO question are bad, because imports might take more time, much more than couple microseconds.

So I'm fine with `sys.version_info >= (3, 9)` because import `sys` costs nothing, it should be already imported somewhere before

Reply:
I agree somewhat. Although for exceptions it is more like 20 microseconds. Imports are cached though and exceptions are not. Nevertheless given the architecture of serde Id rather not do it.